pkg/apb/client.go

```diff
@@ -56,52 +55,23 @@ type Client struct {
 	log           *logging.Logger
 }
 
-func createClientConfigFromFile(configPath string) (*restclient.Config, error) {
-	clientConfig, err := clientcmd.LoadFromFile(configPath)
-	if err != nil {
-		return nil, err
-	}
-
-	config, err := clientcmd.NewDefaultClientConfig(*clientConfig, &clientcmd.ConfigOverrides{}).ClientConfig()
-	if err != nil {
-		return nil, err
-	}
-	return config, nil
-}
-
 func NewClient(log *logging.Logger) (*Client, error) {
 	dockerClient, err := docker.NewClient(DockerSocket)
 	if err != nil {
 		log.Error("Could not load docker client")
 		return nil, err
 	}
 
-	// NOTE: Both the external and internal client object are using the same
-	// clientset library. Internal clientset normally uses a different
-	// library
-	clientConfig, err := restclient.InClusterConfig()
-	if err != nil {
-		log.Warning("Failed to create a InternalClientSet: %v.", err)
-
-		log.Debug("Checking for a local Cluster Config")
-		clientConfig, err = createClientConfigFromFile(homedir.HomeDir() + "/.kube/config")
-		if err != nil {
-			log.Error("Failed to create LocalClientSet")
-			return nil, err
-		}
-	}
-
-	clientset, err := clientset.NewForConfig(clientConfig)
+	k8s, err := clients.Kubernetes(log)
```

Comment:
I like the simplicity of this. I do not like how the `Client` struct has a bunch of clients in it. Will keep looking at this and other PRs to see if I understand the need for it. I know originally it was
```
type Client struct {
    dockerClient ...
}
```